src/WebJobs.Script/Binding/BlobBinding.cs

```diff
@@ -53,12 +54,14 @@ public BlobBinding(ScriptHostConfiguration config, BlobBindingMetadata metadata,
             await BindStreamAsync(context.Value, Access, context.Binder, runtimeContext);
         }
 
-        public override Collection<CustomAttributeBuilder> GetCustomAttributes()
+        public override Collection<CustomAttributeBuilder> GetCustomAttributes(Type parameterType)
         {
             Collection<CustomAttributeBuilder> attributes = new Collection<CustomAttributeBuilder>();
 
+            FileAccess access = GetAttributeAccess(parameterType);
```

Comment:
This now means that the Access property and the actual value used can be out of sync.